### DISCUSSION TITLE
How to deal with "signal 11"?

### DISCUSSION THREAD

**Discussion by dongcongcong606** [idaholab/moose#18344] [Q&A Modules: Phase field] [answered]
Dear Friends!
The following warning appears when I execute the compiled code. And I tried to deal with this problem according to the "Q&A", but I failed. Any friends or experts can offer some experience or suggestions to solve this problem? Thank you very much
"YOUR APPLICATION TERMINATED WITH THE EXIT STRING: Segmentation fault (signal 11)"
![Uploading image.png…]()

**Comment by WilkAndy** [comment, marked as answer]
You'll probably have to give more details - eg, what executable you're using, what input file you're running and what mesh you're using.  All I see is your text and then the string "Uploading image.png..."

**Reply by GiudGiud** [reply]
Could you please share with us a backtrace generated using a debug-mode executable?
See instructions for debugging here
https://mooseframework.inl.gov/application_development/debugging.html
